static/js/popovers.js

```diff
@@ -81,9 +81,21 @@ function show_message_info_popover(element, id) {
             private_message_class: "respond_personal_button",
         };
 
-        var ypos = elt.offset().top - message_viewport.scrollTop();
```

Comment:
`message_viewport.scrollTop()` always returns values greater than `elt.offset().top` once the user scrolls down the initial messages that was loaded on the screen. So `ypos` was always negative. Hence the popup was always placed in bottom.